zenduty/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "0f2dea25-5757-477c-ad92-d459133d8b05",
+  "app_id": "zenduty",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Use Zenduty as the incident response and notification partner for Datadog alerts",
+    "title": "Zenduty",
+    "media": [],
+    "classifier_tags": [
+      "Supported OS::Linux",
+      "Supported OS::macOS",
+      "Supported OS::Windows",
+      "Category::Alerting",
+      "Category::Issue Tracking",
+      "Category::Collaboration",
+      "Category::Notification",
+      "Category::Monitoring"
+    ]
+  },
+  "author": {
+    "support_email": "shubham@zenduty.com",
+    "homepage": "https://github.com/DataDog/integrations-extras",
+    "sales_email": "shubham@zenduty.com",
+    "name": "Zenduty"
+  },
+  "oauth": {},
+  "assets": {
+    "integration": {
+      "source_type_name": "Zenduty",
+      "configuration": {},
+      "events": {
+        "creates_events": false
```

Comment:
In the README it mentions there are events for the integration. If there are events, this should be set to true

Reply:
I assume this refers to whether events are created inside Datadog via Zenduty, which is not a functionality yet, so can be set to false. Let me know if I have this wrong and will make the required changes in that case!